dbms/src/Storages/MergeTree/MergeTreeDataPartChecksum.h

```diff
@@ -100,6 +100,15 @@ struct MinimalisticDataPartChecksums
     uint128 hash_of_uncompressed_files {};
     uint128 uncompressed_hash_of_compressed_files {};
 
+    bool operator==(const MinimalisticDataPartChecksums & other) const
```

Comment:
FYI. You can also provide `tuple` method, that will use `std::tie` or `std::forward_as_tuple`.